v2/pkg/operator/local_stored_collector.go

```diff
@@ -98,24 +101,28 @@ func (o *LocalStorageCollector) OperatorImageCollector(ctx context.Context) ([]v
 		//read the link to the manifest
 		if len(oci.Manifests) == 0 {
 			return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] no manifests found for %s ", op.Catalog)
-		} else {
-			if !strings.Contains(oci.Manifests[0].Digest, "sha256") {
-				return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] the disgets seems to incorrect for %s ", op.Catalog)
-			}
 		}
-		manifest := strings.Split(oci.Manifests[0].Digest, ":")[1]
-		o.Log.Info("manifest %v", manifest)
+		validDigest, err := digest.Parse(oci.Manifests[0].Digest)
+		if err != nil {
+			return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] the disgests seem to incorrect for %s: %v ", op.Catalog, err)
+		}
 
+		manifest := validDigest.Hex()
```

Comment:
Hi @sherine-k,

It seems the Hex() func is deprecated. It is saying to use Digest.Encoded instead